webapp/src/plugin_hooks.ts

```diff
@@ -30,7 +30,8 @@ export default class Hooks {
             return Promise.resolve({message, args: contextArgs});
         }
 
-        if (message.startsWith(createEventCommand)) {
+        const providerConfiguration = await getProviderConfiguration(this.store.getState());
```

Comment:
`getProviderConfiguration` is a synchronous function so no need to await here

```suggestion
        const providerConfiguration = getProviderConfiguration(this.store.getState());
```